doc/API.md

```diff
@@ -3,10 +3,13 @@
 revaultd exposes a [JSON-RPC 2.0](https://www.jsonrpc.org/specification)
 interface over a Unix Domain socket.
 
-| Command                     | Description                        |
-| --------------------------- | ---------------------------------- |
-| [`getinfo`](#getinfo)       | Display general information        |
-| [`listvaults`](#listvaults) | Display a paginated list of vaults |
+| Command                     | Description                               |
+| --------------------------- | ----------------------------------------- |
+| [`getinfo`](#getinfo)       | Display general information               |
+| [`getvault`](#getvault)     | Display specific information of a vault   |
+| [`listvaults`](#listvaults) | Display a paginated list of vaults        |
```

Comment:
I think `getvault` can just be a special case of `listvaults` by using an optional txid parameter.
```
listvaults [vault_txid]
```

Reply:
My idea with `getvault` command is to have a command which retrieve more information than the `listvaults` command for a specific vault.